scripts/tests/chiptest/python_yaml_tester.py

```diff
@@ -0,0 +1,59 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import click
+
+from chip import ChipDeviceCtrl
+from chip.ChipStack import *
+import chip.FabricAdmin
+import chip.CertificateAuthority
+import chip.native
+import chip.yaml
+
```

Comment:
The goal of this PR is to allow both models to run in parallel, until abstraction can be done.

Since it is not REQUIRED to run with the python YAML support, this should be no-op for folks who do not want to run using that, and thus it breaks nobody...

Reply:
I think we have to work on coordinating yaml processing efforts together. 
Unfortunately both the WIP and the existing code process the same thing, but in severly diverge ways. let's discuss this out of the PR since I imagine the back and forth would be longer than what github comments were meant for. 

Sooner is better.